faker/providers/person/__init__.py

```diff
@@ -205,11 +206,17 @@ def name(self) -> str:
         pattern: str = self.random_element(self.formats)
         return self.generator.parse(pattern)
 
-    def first_name(self) -> str:
-        return self.random_element(self.first_names)
+    def first_name(self, min_length: Optional[int] = None, max_length: Optional[int] = None) -> str:
+        return self.random_element(self.first_names, min_length, max_length)
 
-    def last_name(self) -> str:
-        return self.random_element(self.last_names)
+    def first_name_as_list(self) -> Union[tuple, OrderedDict, dict]:
```

Comment:
I think we can get a little bit more specific withthe type hint here:
```suggestion
    def first_name_as_list(self) -> Union[Tuple[str, ...], OrderedDict[str, float], Dict[str, float]]:
```

Reply:
For all similar methods correct?  (e.g. `suffix_female_as_list()`)

Reply:
correct

Reply:
Done